### PR TITLE
schema versions fixes and version upgrades

### PR DESCRIPTION
**Pull request by madhana**

### CODE REVIEW

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            "type": "string"
+        },
+        "adminPassword": {
+            "type": "securestring"
+        },
+        "imagePublisher": {
+            "type": "string",
+            "defaultValue": "Canonical",
+            "metadata": {
+                 "Description": "Image Publisher"
```
Comment (johngossman) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: Mix of "Description" and "description" in this template

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+              "tomcat6",
+			  "tomcat7",
+			  "tomcat8"
+            ]
+		},
+        "location": {
+            "type": "String",
+            "defaultValue" : "West US",
+            "allowedValues": [
+              "West US"
```
Comment (johngossman) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: Did you intend to only allow one region?

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+      "location": "[parameters('location')]",
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "[variables('addressPrefix')]"
+          ]
+        },
+        "subnets": [
+          {
+            "name": "[variables('subnet1Name')]",
```
Comment (johngossman) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: Why 2 subnets?

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+        "dependsOn": [
+            "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+        ],
+        "properties": {
+            "publisher": "Microsoft.OSTCExtensions",
+            "type" : "CustomScriptForLinux",
+            "typeHandlerVersion": "1.2",
+            "settings" : {
+				"fileUris": [
+                        "https://raw.githubusercontent.com/madhana/azure-quickstart-templates/master/openjdk-tomcat-ubuntu-vm/java-tomcat-install.sh"
```
Comment (johngossman) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: Update url to the Azure github?

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json (hunk 1 above)
Comment (madhana) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: fixed

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json (hunk 3 above)
Comment (madhana) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: good catch, left over from previous author, cleaned it up

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json (hunk 4 above)
Comment (madhana) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: done.

File: openjdk-tomcat-ubuntu-vm/azuredeploy.json (hunk 2 above)
Comment (madhana) [inline openjdk-tomcat-ubuntu-vm/azuredeploy.json]: added allowed values

**Pull request merged**